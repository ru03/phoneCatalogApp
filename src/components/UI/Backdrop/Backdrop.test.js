import React from 'react';
import { shallow } from 'enzyme';
import Backdrop from './Backdrop';

describe('Backdrop test', () => {
    it('Backdrop component should be displayed', () => {
        const backdrop = shallow(<Backdrop></Backdrop>);

        expect(backdrop).toMatchSnapshot();
    });
})