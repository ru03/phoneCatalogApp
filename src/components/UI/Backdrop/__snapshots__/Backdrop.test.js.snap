// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop test Backdrop component should be displayed 1`] = `
<div
  className="Backdrop"
/>
`;
